libs/features/personalization/personalization.js

```diff
@@ -540,6 +540,27 @@ export function parseManifestVariants(data, manifestPath, targetId) {
   return null;
 }
 
+function createMartechMetadataBlock(placeholders, config) {
+  if (config.locale.ietf === 'en-US') return;
+
+  const section = createTag('div');
+  const metaEl = createTag('div', { class: 'martech-metadata' });
+
+  section.append(metaEl);
+
+  const firstRow = placeholders[0];
+  const usCol = firstRow['en-us'] || firstRow.us || firstRow.en || firstRow.key;
+
+  if (!usCol) return;
+
+  Object.values(config.placeholders).forEach((item) => {
+    const row = createTag('div', undefined, `<div>${item}</div><div>${usCol}</div>`);
+    metaEl.appendChild(row);
+  });
+
+  document.querySelector('main').appendChild(section);
```

Comment:
Is there any reason to store the data in the html? I'd expose it on `window.X` rather than adding markup. I feel like it's somewhat of an anti-pattern to store things within the HTML unless there's a good reason to do so